test-tools/testcafe.ts

```diff
@@ -0,0 +1,73 @@
+import { Selector } from 'testcafe';
+
+fixture `SFX Web Components Functional Testing`
+      .page `../demo/demo.html`;
+
+const searchbox = Selector('sfx-search-box#main-search');
+const input = Selector('sfx-search-box#main-search > input');
+const clearButton = Selector('.sfx-clear');
+const autocomplete = Selector('sfx-autocomplete');
+
+test('SFX typing in searchbox populates sayt terms and sayt products', async t => {
+  // confirm searchbox exists
+  await t.expect(searchbox).ok('Searchbox exists')
+  await t.hover(searchbox)
+  await t.click(searchbox)
+  // confirm searchbox input
+  await t.typeText(searchbox, 'check dress')
+  await t.expect(input.value).contains('check dress', 'Searchbox has successful input value')
+  // confirm autocomplete terms are present
+  const autocompleteTermsList = autocomplete.child('ul');
+  await t.expect(await autocompleteTermsList.childElementCount).gte(2, 'Autocomplete terms are present')
+  // confirm autocomplete products are present
+  const autocompleteProductsList = Selector('sfx-products-sayt');
+  await t.expect( await autocompleteProductsList.childElementCount).gte(2, 'Autocomplete products are present')
+
+});
+
+test('SFX autcomplete terms has hover functionality', async t => {
+  // confirm searchbox exists
+  await t.expect(searchbox).ok('Searchbox exists')
+  await t.hover(searchbox)
+  await t.click(searchbox)
+  // confirm searchbox input
+  await t.typeText(searchbox, 'check dress')
+  await t.expect(input.value).contains('check dress', 'Searchbox has successful input value')
+  // confirm user can hover over autocomplete terms
+  const autocompleteTermIndex = 2;
+  const thirdAutocompleteTerm = await autocomplete.child('ul').child('li').nth(autocompleteTermIndex)();
```

Comment:
I think you can just call this constant `autocompleteTerm`, unless you disagree.